Bender.yml

```diff
@@ -1,14 +1,19 @@
 package:
   name: hci
-  authors: 
+  authors:
     - "Francesco Conti <f.conti@unibo.it>"
     - "Gianna Paulin <pauling@iis.ee.ethz.ch"
     - "Tobias Riedener <tobiasri@student.ethz.ch>"
+    - "Luigi Ghionda <luigi.ghionda2@studio.it>"
+    - "Arpan Suravi Prasad <prasadar@iis.ee.ethz.ch>"
 
 dependencies:
   hwpe-stream:            { git: "https://github.com/pulp-platform/hwpe-stream.git", version: 1.8.0 }
   cluster_interconnect:   { git: "https://github.com/pulp-platform/cluster_interconnect.git", rev: 1284def6c0b7f7e9355eb093d00883ad9dead1b7 }
   L2_tcdm_hybrid_interco: { git: "https://github.com/pulp-platform/L2_tcdm_hybrid_interco.git", version: 1.0.0 }
+  redundancy_cells:       { git: "https://github.com/pulp-platform/redundancy_cells.git", rev: c37bdb47339bf70e8323de8df14ea8bbeafb6583 } # branch: astral_rebase
+  common_cells:           { git: "https://github.com/pulp-platform/common_cells.git", version: 1.24.0 }
+  register_interface:     { git: "https://github.com/pulp-platform/register_interface.git", version: 0.3.1 }
```

Comment:
do we actually *need* this or is it only for the parametric types `hci_ecc_req_t`?